test/e2e/case17_policy_webhook_test.go

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2020 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package e2e
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"open-cluster-management.io/governance-policy-propagator/test/utils"
+)
+
+const (
+	case17PolicyLongYaml string = "../resources/case17_policy_webhook/case17_policy_long.yaml"
+	longNamesapce        string = "long-long-long-long-long-long-long"
+	errMsg               string = `admission webhook "policy.open-cluster-management.io.webhook" denied the ` +
+		`request: the combined length of namespace and policy name (namespaceName.policyName) ` +
+		`should not exceed 63 characters`
+)
+
+var _ = Describe("Test policy webhook", Label("webhook"), func() {
+	BeforeEach(func() {
+		_, err := utils.KubectlWithOutput("create",
+			"ns", longNamesapce,
+		)
+		Expect(err).ShouldNot(HaveOccurred())
+	})
+	AfterEach(func() {
+		// cleanup
+		_, err := utils.KubectlWithOutput("delete",
+			"ns", longNamesapce,
+			"--ignore-not-found",
+		)
+		Expect(err).ShouldNot(HaveOccurred())
+	})
+	Describe("Test name + namespace over 63", func() {
+		It("Should the error message is presented", func() {
```

Comment:
Can we add another test for when the error doesn't happen? Since this is the only test running right now when the webhook is enabled, I think?

Reply:
I tried to add update case but namespace and name cannot be modified... I tried to add long namespace and short name but namespace itself has limitation so... would you give some idea?!